Assets/Resources/Dialogue/tutorial-movement.xml

```diff
@@ -0,0 +1,8 @@
+<dialogue>
+    <name>Movement</name>
+    <sentences>
+        <value>Welcome to the tutorial!</value>
+        <value>To walk around, use WASD or arrow keys</value>
```

Comment:
The program is controller compatible. Mention being able to use a joystick to move?